code/modules/mob/living/carbon/xenomorph/abilities/shrieker/shrieker_powers.dm

```diff
@@ -0,0 +1,219 @@
+/datum/action/xeno_action/activable/shriek/use_ability(atom/A)
+	var/mob/living/carbon/Xenomorph/xeno = owner
+	if (!istype(xeno))
+		return
+
+	if (!action_cooldown_check())
+		return
+	if (!check_plasma_owner())
```

Comment:
Do we ever use plasma here? it never is use_plasma'ed or the check_and_use proc.

Reply:
yea it uses quite a bit of plasma